RELEASE_NOTES.md

```diff
@@ -7,9 +7,9 @@ This file contains a high-level description of this package's evolution. Release
 
 ## 4.9.2 - TBD
 
-
-* Update `nc-config` to remove inclusion from automatically-detected `nf-config` and `ncxx-config` files, as the wrong files could be included in the output.  This is in support of [GitHub #2274](https://github.com/Unidata/netcdf-c/issues/2274).
+* [Ehancement] Update H5FDhttp.c to use the newly registered standard VFD identifier. See [Github #????](https://github.com/Unidata/netcdf-c/pull/????).
```

Comment:
Do we have a github number for this instead of ????

Reply:
Sorry, I forgot to add it; PR is 2629